client/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
we already have `.gitignore` file in the root
I think that will be enough

Reply:
True, we don't need `.gitignore` in the `client` folder,
@amusameh , you can add anything you want to ignore there in the root

Reply:
this is the react .gitingore I don't see any reason to delete it since it's a separate server